mailbag/controller.py

```diff
@@ -112,4 +117,8 @@ def generate_mailbag(self):
                         writer = csv.writer(f)
                         writer.writerows(portion)
 
+        #Loading the bag and saving manifests
+        bag = bagit.Bag(mailbag_dir)
+        bag.save(manifests=True)
```

Comment:
You get `bag` on line 54, so I don't think you need to re-load it with `bagit.Bag()`. Instead of that line, you should check `if not self.args.dry_run` again before `bag.save(manifests=True)` and that should work.